tests_integration/runPrettier.js

```diff
@@ -71,9 +71,6 @@ function runPrettier(dir, args, options) {
   // We cannot use `jest.setMock("get-stream", impl)` here, because in the
   // production build everything is bundled into one file so there is no
   // "get-stream" module to mock.
-  jest.spyOn(require(thirdParty), "getStream").mockImplementation(() => ({
```

Comment:
The reason for this change was that I wanted to sort third-party scripts alphabetically. I reverted that in https://github.com/prettier/prettier/pull/4192/commits/bfbc580af2f15851aff75b59491e03c259ccc718, but I'd be keen to delete it and thus to keep sorting. This will help read and write other third-party scripts in future. WDYT?